Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:wheezy
+
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get install -y python-pip python-virtualenv virtualenvwrapper python-dev libjpeg-dev libxml2-dev libxslt1-dev zlib1g-dev build-essential gettext
```

Comment:
Yes; when done, we will need to make docker forget about caches when building the image to take updated deps into account.